src/main/kotlin/org/jitsi/nlj/transform/node/outgoing/OutgoingStatisticsTracker.kt

```diff
@@ -37,15 +37,18 @@ class OutgoingStatisticsTracker : ObserverNode("Outgoing statistics tracker") {
         stats.packetSent(rtpPacket.length, rtpPacket.timestamp)
     }
 
-    override fun getNodeStats(): NodeStatsBlock {
-        return super.getNodeStats().apply {
-            val stats = getSnapshot()
-            stats.ssrcStats.forEach { (ssrc, streamStats) ->
-                addJson(ssrc.toString(), streamStats.toJson())
-            }
+    override fun getNodeStats(): NodeStatsBlock = super.getNodeStats().apply {
+        val stats = getSnapshot()
+        stats.ssrcStats.forEach { (ssrc, streamStats) ->
+            addJson(ssrc.toString(), streamStats.toJson())
         }
     }
 
+    /**
+     * Don't aggregate the per-SSRC stats.
+     */
+    override fun getNodeStatsToAggregate(): NodeStatsBlock = super.getNodeStats()
```

Comment:
Should the `super` call `getNodeStatsToAggregate()` here?  (They happen to be the same thing in this case, but it's probably good form.)